scrapper/script.py

```diff
@@ -227,9 +270,13 @@ def scrap_meta_table(region, db_path, start_date, end_date, history):
         elif total_records <= total_records_in_db:
             print('Skipping range %s : %s because total_records[%s] <= total_records_in_db[%s]' % (start_date, end_date, total_records, total_records_in_db))
             return
-
+        elif total_records == total_records_before_filter:
+            # handle unresponsive gisaid
+            print('Number of records didn\'t changed after filtering by date')
+            return 
```

Comment:
In case of unresponsive gisaid we should raise an exception. Parent function will then retry running this function.